src/System.Private.CoreLib/src/System/Environment.CoreCLR.cs

```diff
@@ -83,8 +83,6 @@ public static string[] GetCommandLineArgs()
         // Unconditionally return false since .NET Core does not support object finalization during shutdown.
         public static bool HasShutdownStarted => false;
```

Comment:
You can move this one to the shared partition too while you are on it.

Reply:
Done